setup.py

```diff
@@ -6,19 +8,18 @@
 
 
 def get_install_requires():
+    """Get requirements for adviser module."""
     with open("requirements.txt", "r") as requirements_file:
         # TODO: respect hashes in requirements.txt file
         res = requirements_file.readlines()
         return [req.split(" ", maxsplit=1)[0] for req in res if req]
 
 
-def get_test_requires():
-    with open("requirements-test.txt", "r") as requirements_file:
-        res = requirements_file.readlines()
-        return [req.split(" ", maxsplit=1)[0] for req in res if req]
+get_test_requires = get_install_requires
```

Comment:
Don't we need to get `requirements-test.txt` here instead of `requirements.txt`?